pages/checkin.tsx

```diff
@@ -168,20 +169,18 @@ export default function CheckinPage() {
           </Text>
           <Box height={1} />
           <Text>
+            <p>{introText}</p>
             <p>
-              {isCareEnv
-                ? 'Durch die aktuellen Corona-Verordnungen müssen Sie Ihre Kontaktdaten hinterlegen, wenn Sie in einem Betrieb sind der zu Schutzmaßnahmen verpflichtet ist, wie z.B Pflegeeinrichtungen. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.'
-                : 'Durch die aktuellen Corona-Verordnungen musst du deine Kontaktdaten hinterlegen, wenn Du in einem Betrieb bist der zu Schutzmaßnahmen verpflichtet ist, wie z.B Restaurants. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.'}
-            </p>
-            <p>
-              {isCareEnv
+              {isCareEnv || isFreseniusEnv
                 ? 'So kann das Gesundheitsamt Sie anrufen, wenn es notwendig ist.'
                 : 'So kann das Gesundheitsamt Dich anrufen, wenn es notwendig ist.'}
             </p>
             <p>
-              Datenschutz ist uns dabei sehr wichtig! <strong>recover</strong>{' '}
-              speichert {isCareEnv ? 'Ihre' : 'Deine'} Daten verschlüsselt und
-              sicher.
+              Datenschutz ist uns dabei sehr wichtig!{' '}
+              {isRcvrEnv
+                ? <strong>recover</strong> +
+                  ' speichert Deine Daten verschlüsselt und sicher.'
```

Comment:
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/2098462/92222373-371bc900-ee9f-11ea-9f11-1ac4baaf753a.png">

You can't use `+` here.

```suggestion
                ? <><strong>recover</strong> speichert Deine Daten verschlüsselt und sicher.</>
```

Reply:
I have no Idea why this happened. 🤔